Documentation/generics.md

```diff
@@ -0,0 +1,64 @@
+*.NET Innovation Examples*
+**GENERICS**
+===================
+
+We use generics implicitly all the time in  C#. When you use LINQ in C#, do you notice you are working with <code>IEnumerable\<T\></code>? Ever wonder what <code>T</code> is and what it means?
```

Comment:
You can just wrap the statements in single backticks \` instead of wrapping them in `<code>` tags, then you can get rid of escaping the `< >` as well.